src/factories/posts.ts

```diff
@@ -0,0 +1,35 @@
+import { Posts } from '../models/posts'
+import { ServerPostArray } from '../interfaces/serverPostArray'
+import { ServerUserElement } from '../interfaces/serverUserElement'
+
+export const convertPosts = (
```

Comment:
Let's create this as a factory file. For example:

```
export const postsFactory = {
    convertPosts(...) = {}
}
```

`postsFactory.convertPosts(...)`